pkgs/applications/misc/polar-bookshelf/default.nix

```diff
@@ -0,0 +1,98 @@
+{ stdenv, lib, makeWrapper, fetchurl
+, dpkg, wrapGAppsHook
+, gtk3, cairo, gnome2, atk, gdk_pixbuf, glib
+, at-spi2-atk, dbus, libX11, libxcb, libXi
+, libXcursor, libXdamage, libXrandr, libXcomposite
+, libXext, libXfixes, libXrender, libXtst, libXScrnSaver
+, nss, nspr, alsaLib, cups, fontconfig, expat
+, libudev0-shim, glibc, curl, openssl, nghttp2, gsettings_desktop_schemas }:
+
+
+stdenv.mkDerivation rec {
+  name = "polar-bookshelf-${version}";
+  version = "1.0.2";
+
+  src = fetchurl {
+    url = "https://github.com/burtonator/polar-bookshelf/releases/download/${version}/polar-bookshelf-${version}-amd64.deb";
```

Comment:
why fetch a deb? I suppose because it's a electron app, but at least document it.

Reply:
Sure thing.